python/lsst/ts/fbs/utils/maintel/sv_surveys.py

```diff
@@ -570,8 +580,9 @@ def gen_template_surveys(
         if bandname in dark_only:
             bfs.append((bf.MoonAltLimitBasisFunction(alt_limit=-5), 0.0))
 
-        # limit to first year
-        bfs.append((bf.OnlyBeforeNightBasisFunction(night_max=366), 0.0))
+        # limit to first year -
+        # BUT for SV surveys we know < 1 year, plus "night" is wrong
```

Comment:
Do we plan on re-introducing this at some point? If not, can we just remove these lines instead of commenting them out?

Reply:
I would like to reintroduce it in general .. I suppose for SV surveys, it could be dropped entirely but I'm a kind of hoping we can take these configs (as they are) and update them for on-sky main survey, as they are very very similar.
I would also like to put it in once we get "night" sorted out, so we can make sure it's working as expected.